src/plugins/intel_cpu/CMakeLists.txt

```diff
@@ -33,8 +33,13 @@ else()
 endif()
 set(OV_CPU_ARM_TARGET_ARCH ${OV_CPU_ARM_TARGET_ARCH_DEFAULT})
 
-# enbale mlas for X86 cpus only
-ie_dependent_option(ENABLE_MLAS_FOR_CPU "MLAS GEMM for OpenVINO CPU Plugin" ON "X86 OR X86_64" OFF)
+if(X86 OR X86_64)
+    set(ENABLE_MLAS_FOR_CPU_DEFAULT ON)
+else()
+    set(ENABLE_MLAS_FOR_CPU_DEFAULT OFF)
```

Comment:
Since we already enable MLAS Transpose as first priority we can set ON by default for ARM as well. For other architectures (like RV64) it should be OFF.